### PR TITLE
Deis release 1.3

### PR DESCRIPTION
**Pull request by mboersma**

### CODE REVIEW

**Comment by bacongobbler** [comment]
There was a really cool post by @holman two weeks ago on how they do blog post style testing at github recently. We should learn from their experience and do some styling tests of our own at some point :)

http://zachholman.com/posts/how-github-writes-blog-posts/

File: _posts/2015-02-03-deis-1-3-continuous-improvement.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+meta:
+  - name: description
+    content: Deis releases v1.3 with a continuous flow of fixes and improvements.
+  - name: keywords
+    content: deis, release, v1.3, stable, production, production-ready, paas, private paas, heroku, github, docker, coreos, opdemand, enterprise
+---
+
+The Deis project is proud to announce [v1.3.0](https://github.com/deis/deis/releases/tag/v1.3.0), which includes more ways to debug your cluster, code quality improvements to the controller and security enhancements to the builder.
+
+**Important:** This release includes a CoreOS update for the CVE-2015-0235 glibc fix. All Deis users are encouraged to upgrade their host machines to CoreOS 522.6.0 or later.
```
Comment (mboersma) [inline _posts/2015-02-03-deis-1-3-continuous-improvement.md]: Link CVE to http://web.nvd.nist.gov/view/vuln/detail?vulnId=CVE-2015-0235 ?

File: _posts/2015-02-03-deis-1-3-continuous-improvement.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+- Vagrant VMs default to 2GB, since less can lead to docker mem alloc errors
+- Dockerfiles include `ENV DEIS_RELEASE` to prevent overlap in `docker images`
+- `etcd` discovery URL uses the right key on GCE
+- a Procfile can override a buildpack's `default_process_types`
+- [controller](https://github.com/deis/deis/blob/master/controller) tracks create and start separately for better error handling
+- [controller](https://github.com/deis/deis/blob/master/controller) consistently returns JSON errors
+- [controller](https://github.com/deis/deis/blob/master/controller) validates uploaded key material
+- [logger](https://github.com/deis/deis/blob/master/logger) builds with `godep`, like other Go components
+
```
Comment (mboersma) [inline _posts/2015-02-03-deis-1-3-continuous-improvement.md]: We should add:

**Comment by mboersma** [comment]
If we're publishing this today, let's not forget to change the title of the post since it contains the date.

**Comment by mboersma** [comment]
I'm ok with this very boring title. We will need to update the markdown filename to match. :green_heart:

**Comment by bacongobbler** [comment]
I've addressed both

> let's not forget to change the title of the post since it contains the date.

and

> We will need to update the markdown filename to match.

So we're ready to go today if we're ready :)

**Comment by mboersma** [comment]
:+1:

**Comment by carmstrong** [comment]
LGTM. Wish we had a better title, but I'm not coming up with anything compelling.

**Comment by gabrtv** [comment]
Let's go with "Deis v1.3 - Code Quality & Security Enhancements".  I also provided @mboersma with new language around scheduler prototyping.

**Pull request merged**